games/devtest/mods/experimental/commands.lua

```diff
@@ -43,6 +43,79 @@ minetest.register_chatcommand("test_bulk_set_node", {
 	end,
 })
 
+-- Safeguard against too much optimization.
+_G._bench_content_ids_data = setmetatable({}, {__mode = "k"})
```

Comment:
what does this do again?

Reply:
I just want to make sure LuaJIT doesn't see that the result is unused and throw away the entire benchmark. I doubt it would do that much optimization anyway, but it's a precaution.

Reply:
I meant the `mode = "k"` thing, weak keys?
in any case please write it down in a comment

Reply:
Comment added.